zkp/golang/internal/key-manager/key/key-manager.go

```diff
@@ -0,0 +1,33 @@
+// Copyright © 2024 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package key
+
+import (
+	"github.com/hyperledger-labs/zeto/pkg/key-manager/core"
+	"github.com/iden3/go-iden3-crypto/babyjub"
+)
+
+func NewKeyEntryFromPrivateKeyBytes(keyBytes [32]byte) *core.KeyEntry {
+	var privKey babyjub.PrivateKey
+	copy(privKey[:], keyBytes[:])
+	privKeyForZkp := babyjub.SkToBigInt(&privKey)
+	return &core.KeyEntry{
+		PrivateKey:       &privKey,
+		PublicKey:        privKey.Public(),
+		PrivateKeyForZkp: privKeyForZkp,
+	}
+}
```

Comment:
@jimthematrix I think we should just replace the `pkg` key-manager.go with this file for now along with its test.